exercises/Sql-3-R.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: exercise
+title: R-SQL 3
+subtitle: Data Stream
+language: R
+---
+
+This is a follow-up to [R-SQL 2]({{ site.baseurl }}/exercises/Sql-2-R).
+
+Dr. Undómiel is really testing your kindness now. She's seen that all of the 
+average hind foot length and average weights are pretty similar within species, 
+but she wants to support the idea with some statistics. 
+
+1. Modify your species list to remove `species_id` with insufficient data 
+(*i.e., remove species that return <NA> in the R-SQL 2 results table for either `sex` AND remove `species=='PX'` because it only has one entry per `sex` *). 
```

Comment:
Also, I think the better solution here is to add a check for >=2 species inside the loop.